ci/util/cmd.go

```diff
@@ -0,0 +1,25 @@
+package util
+
+import (
+	"os"
+	"path"
+
+	"github.com/magefile/mage/sh"
+)
+
+// GetGoBinaryPath looks for the given binary in path, if not checks if it's in $GOPATH/bin
+func GetGoBinaryPath(binaryName string) (string, error) {
+	res, err := sh.Output("which", binaryName)
+	if err != nil {
+		return "", err
+	}
+	if res != "" {
+		return res, nil
+	}
+	gopath := os.Getenv("GOPATH")
```

Comment:
`GOPATH` variable can be empty starting from 1.8 https://golang.org/doc/go1.8#gopath
In this case, it has a default value:`$HOME/go`

Reply:
forgot to mention - fixd